### ISSUE TITLE
latex.fmt doesn't match pdftex.pool

### ISSUE THREAD

**Issue by swenzel** [manuels/texlive.js#41]
When trying to create a pdf file I get this error:
```
This is pdfTeX, Version 3.1415926-1.40.11 (TeX Live 2010)
 restricted \write18 enabled.
---! //texmf-var/web2c/pdftex/latex.fmt doesn't match pdftex.pool
(Fatal format file error; I'm stymied)
```
I have no idea what to do or how to create the latex.fmt myself, do you know this problem?

**Comment by manuels** [comment]
You probably have recompiled the project, right?
Then you must use the newly compiled latex.fmt and pdftex.pool.
I also experienced some strange issues with caching in Firefox (I think you can try the 'private mode' to check if this is an issue).

**Comment by fzimmermann89** [comment]
Can you elaborate on the caching issues?

2016-03-18 9:54 GMT+01:00 manuels <notifications@github.com>:

**Comment by fzimmermann89** [comment]
A note on recompiling: You will need emscripten 1.35.5 or newer

2016-03-18 12:58 GMT+01:00 Felix Zimmermann <fzimmermann89@gmail.com>:

**Comment by manuels** [comment]
> Can you elaborate on the caching issues?

Well, I experienced some strange behavior a while a go:
I got this error in Firefox (my default browser) but not in Chrome using exactly the same setup. Then I used the private mode in Firefox and it worked there, too. So I cleared the Firefox cache and I got rid of this error.

**Comment by swenzel** [comment]
Yes I did recompile it, but I've also tried the original version. I linked the texlivejs directory in my project, therefore it is using the very same files which are also used when I open the index.html from texlivejs directly.
I saw that the size of the retrieved `latex.fmt` differs quite a lot from the proper one so it seems to be indeed a caching issue. But it is a very strange one. Neither a private window nor clearing the cache manually and restarting firefox solves it. In Chrome everything works but I run into memory limitations (which is a different story).
